src/main/java/rwg/support/Support.java

```diff
@@ -39,9 +40,18 @@ public static void init() {
             SupportEBXL.init();
         }
 
-        if (Loader.isModLoaded("Thaumcraft")) {
-            SupportTC.init();
+        if (ConfigRWG.suptcbig) {
+            if (Loader.isModLoaded("Thaumcraft")) {
+                SupportTC.init();
+            }
+        } else if (Loader.isModLoaded("Thaumcraft")) {
+            SupportTC1.init();
+        }
+
```

Comment:
No need in condition here. Also there's no need in additional class. This logic can be implemented inside exisiting one.
```suggestion
        if (Loader.isModLoaded("Thaumcraft")) {
            SupportTC.init();
        }
```